### PR TITLE
Added Talk Python Episode 313 to docs

### PR DESCRIPTION
**Pull request by RatulMaharaj**
## Change Summary

Add episode 313 of the *Talk Python To Me* podcast, where Micheal Kennedy and Samuel Colvin discuss *pydantic*, to the docs.

## Related issue number

No related issues.

## Checklist

* [x] Unit tests for the changes exist
* [x] Tests pass on CI and coverage remains at 100%
* [x] Documentation reflects the changes where applicable
* [x] `changes/<pull request or issue id>-<github username>.md` file added describing change
  (see [changes/README.md](https://github.com/samuelcolvin/pydantic/blob/master/changes/README.md) for details)

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -121,6 +121,9 @@ For a more comprehensive list of open-source projects using *pydantic* see the
 
 Podcasts and videos discussing pydantic.
 
+[Talk Python To Me](https://talkpython.fm/episodes/show/313/automate-your-data-exchange-with-pydantic){target=_blank}
+: Micheal Kennedy and Samuel Colvin, the creator of *pydantic*, dive into the history of pydantic and it's many uses and benefits.
```
Comment (PrettyWood) [inline docs/index.md:125]:
```suggestion
: Micheal Kennedy and Samuel Colvin, the creator of *pydantic*, dive into the history of pydantic and its many uses and benefits.
```

**Review by PrettyWood — CHANGES_REQUESTED** [review]

**Comment by PrettyWood** [comment]
Please update and rebase/merge on master. Other than that LGTM thanks!

**Comment by RatulMaharaj** [comment]
I've updated and rebased.

File: docs/index.md [hunk 2]
```diff
@@ -121,6 +121,9 @@ For a more comprehensive list of open-source projects using *pydantic* see the
 
 Podcasts and videos discussing pydantic.
 
+[Talk Python To Me](https://talkpython.fm/episodes/show/313/automate-your-data-exchange-with-pydantic){target=_blank}
+: Micheal Kennedy and Samuel Colvin, the creator of *pydantic*, dive into the history of pydantic and its many uses and benefits.
```
Comment (PrettyWood) [inline docs/index.md:125]: Typo in Michael's name in both the doc and the change file

**Review by PrettyWood — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 2 above)
Comment (RatulMaharaj) [reply docs/index.md]: Fixed typo with commit fa2428344f52c65271fd0444032f40f1c81f854f

**Review by PrettyWood — APPROVED** [review]

**Comment by PrettyWood** [comment]
Thanks!

**Pull request merged**